### ISSUE TITLE
outputs and targets being fed in reverse order to loss functions

### ISSUE THREAD

**Issue by AlexFuster** [magenta/ddsp#59]
At the end of the function call of the class Autoencoder in ddsp/training/models.py, we have these lines, where it clearly feeds the loss_obj with the target and the synthetized audios in that order
```python
if training:
  for loss_obj in self.loss_objs:
    self.add_loss(loss_obj(features['audio'], audio_gen))
```
However, in ddsp/losses.py, we see how the headers of all the loss functions implemented are the following
```python
def call(self, audio, target_audio):
```
which means that they expect first the synthetized audio and then the target audio.

So basically, they are fed in reverse order. For most loss functions, this fact is not relevant, that's why this has been unnoticed until now. It is still a bug, though.

**Comment by jesseengel** [comment]
Thanks for pointing this out. Indeed since they are all symmetric it doesn't change any of the current experiments. We'll fix it, or feel free to submit a PR if you want.

**Comment by AlexFuster** [comment]
sure, done

**Comment by jesseengel** [comment]
#64
